packages/did-resolver/index.ts

```diff
@@ -84,6 +85,21 @@ class CardstackIdentifier {
   }
 }
 
+function normalizeUniqueId(candidate: string) {
```

Comment:
Are we using a short unique id for the sake of storage space?

Reply:
To keep the did more compact / less storage space in on-chain scenarios / easier to copy/paste